tests/arginc.list

```diff
@@ -1,11 +1,11 @@
 1: int main(int argc, char **argv)
 2: {
 3:     return argc - 2;
+4: }
```

Comment:
The code listing in file `docs/IR.md` should be adjusted because of the introduction of AST.

Reply:
Mmm.This is such a problem.I will fix it later.

Reply:
ping?